raiden/raiden_service.py

```diff
@@ -325,6 +332,11 @@ def stop(self):
         if self.db_lock is not None:
             self.db_lock.release()
 
+    @property
+    def _pending_transfers(self):
+        results = chain(*self.identifier_to_results.values())
```

Comment:
This is counting the total number of pending transaction across all channels, which is not what we want, we want to limit the number of transactions on a per channel basis.

It's better to use the constant to filter outgoing transfer in these places (here we want to avoid sending transfers which would be reject by the node's partners):

- [initiator.py:next_channel_from_routes](https://github.com/raiden-network/raiden/blob/b3251cb92fe7eb6a39649a08e3b500b0c33fc88d/raiden/transfer/mediated_transfer/initiator.py#L51-L80)
- [mediator.py:next_channel_from_routes](https://github.com/raiden-network/raiden/blob/b3251cb92fe7eb6a39649a08e3b500b0c33fc88d/raiden/transfer/mediated_transfer/mediator.py#L282-L328)

And when receiving a transfer at the channel level (here we want to avoid a balance proof which contains a large number of pending transfer, which the node cannot unlock on-chain because of the gas limitations):

- [channel.py:valid_lockedtransfer_check](https://github.com/raiden-network/raiden/blob/b3251cb92fe7eb6a39649a08e3b500b0c33fc88d/raiden/transfer/channel.py#L389)